src/components/App.js

```diff
@@ -34,7 +38,49 @@ class App extends Component {
     }
 
     console.log(('Starting the app for platform: ' + this.state.os));
+    this.initLinkedIn();
+  }
+
+  // Initialize linkedin tokens
+  initLinkedIn() {
+    let redirecUrl = 'http://localhost';
```

Comment:
![image](https://cloud.githubusercontent.com/assets/8914311/18072435/3d4cdeca-6e99-11e6-8c15-a5c443448daa.png)
Above is comment from author of LinkedIn login module. I think we can assign whatever we want in that variable... It's bit weird but it seems doesn't matter. I'll leave comment about that. Thank you